Jaace/planning-poker
# Create Backend Prisma Scaffolding

## Description
Adds the base setup for using Prisma as the data layer to power this application. Necessary dependencies added and a basic workflow to deploy to Prisma and pull back the generated `prisma.graphql` file for use with the application.

## Test
* Run `prisma deploy` and see that the `datamodel.prisma` gets deployed to the Prisma server.
* See that the post-deploy hook runs and places the `prisma.graphql` file where it should live.
* You'll need to have your `.env` set up for this to work with the `PRISMA_ENDPOINT` and `APP_DEFAULT_ENDPOINT` defined.

## Related Issue
Resolve #1
